site/content/3.12/release-notes/version-3.12/whats-new-in-3-12.md

```diff
@@ -241,5 +241,77 @@ of outgrowing the maximum number of file descriptors the ArangoDB process
 can open. Thus, these options should only be enabled on deployments with a
 limited number of collections/shards/indexes.
 
+## Client tools
+
+### arangodump
+
+#### Improved dump performance
+
+ArangoDB 3.12 includes extended parallelization capabilities to work not only
+at the collection level, but also at the shard level. In combination with the
+new optimized format, database dumps are now created and restored more quickly
+and occupy minimal disk space. This major performance boost makes dumps and
+restores up to several times faster, which is extremely useful when dealing
+with large shards.
+
+The new dump variant can be enabled via `--use-parallel-dump`. The default
+value is `true`.
+
+To achieve the best dump performance and the smallest data dumps in terms of
+size, you can use the `--dump-vpack` option. The resulting dump data is stored
+in velocypack format instead of JSON. The velocypack format is more compact than
+JSON, therefore the output file size can be reduced compared to JSON, even
+when compression is enabled, and can also lead to faster dumps. Note, however,
+that this option is experimental and disabled by default.
```

Comment:
```suggestion
when compression is enabled. It can also lead to faster dumps because there is less data to
transfer and no conversion from the server-internal format (VelocyPack) to JSON is needed.
Note, however, that this option is experimental and disabled by default.
```